modules/installation/proc-rhdh-deploy-gke-helm.adoc

```diff
@@ -0,0 +1,175 @@
+----
+<1> Enter your {gke-short} namespace in the command.
+<2> Enter your username in the command.
+<3> Enter your password in the command.
+<4> Enter your email address in the command.
+
+The created pull secret is used to pull the {product-short} images from the {company-name} Ecosystem.
+--
+
+. Set up a Google-managed certificate by creating a `ManagedCertificate` object that you must attach to the ingress.
++
+--
+.Example of attaching a `ManagedCertificate` object to the ingress
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: <rhdh_certificate_name>
+spec:
+  domains:
+    - <rhdh_domain_name>
+----
+--
+For more information about setting up a Google-managed certificate, see https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs?hl=en#setting_up_a_google-managed_certificate[Setting up a Google-managed certificate].
+
+. Create a `FrontendConfig` object to set a policy for redirecting to HTTPS. You must attach this policy to the ingress. 
++
+--
+.Example of attaching a `FrontendConfig` object to the ingress
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: <ingress_security_config>
+spec:
+  sslPolicy: gke-ingress-ssl-policy-https
+  redirectToHttps:
+    enabled: true
+----
+--
+For more information about setting a policy to redirect to HTTPS, see https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration?hl=en#https_redirect[HTTP to HTTPS redirects].
+
+. Create a file named `values.yaml` using the following template:
++
+.Example `values.yaml` file
+[source,yaml,subs="attributes+"]
+----
+global:
+  host: <rhdh_domain_name>
+route:
+  enabled: false
+upstream:
+  service:
+    type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: gce
+      kubernetes.io/ingress.global-static-ip-name: <ADDRESS_NAME>
+      networking.gke.io/managed-certificates: <rhdh_certificate_name>
+      networking.gke.io/v1beta1.FrontendConfig: <ingress_security_config>
+    className: gce
+  backstage:
+    image:
+      pullSecrets:
+      - rhdh-pull-secret
+    podSecurityContext:
+      fsGroup: 2000
+  postgresql:
+    image:
+      pullSecrets:
+      - rhdh-pull-secret
+    primary:
+      podSecurityContext:
+        enabled: true
+        fsGroup: 3000
+  volumePermissions:
+    enabled: true
+----
+. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the `values.yaml` file:
++
+[source,terminal,subs="attributes+"]
+----
+helm -n <your_namespace> install -f values.yaml <your_deploy_name> \
+  openshift-helm-charts/redhat-developer-hub \
+  --version {product-chart-version}
+----
++
+For the latest Helm Chart version, see this https://github.com/openshift-helm-charts/charts/tree/main/charts/redhat/redhat/redhat-developer-hub[Helm Charts] repository.
++
+
+[NOTE]
```

Comment:
@hmanwani-rh The procedure is about _installing_ GKE, so is uninstalling or upgrading really part of the _verification_ of an _installation_ procedure.

I see your point though, so maybe we should have separate sections for upgrading and deleting your deployment after the installation procedure rather than placing them in the _Verfication_ section of the installation procedure?

Reply:
I think we can certainly include a few commands in the verification section, as deleting and upgrading inherently confirm that something is already installed, which can then be deleted or upgraded. However, if there is substantial content to be added under the deleting or upgrading sections, having a separate procedure would make sense.

Reply:
@hmanwani-rh Moved delete and upgrade commands to Verification section.